webhook-service/lib/template_engine_test.go

```diff
@@ -42,6 +43,46 @@ func TestTemplateEngine_ParseTemplate(t1 *testing.T) {
 			want:    "",
 			wantErr: true,
 		},
+		{
```

Comment:
Is there a way to get any information on what field of a template string was causing the error? I.e. like described in the issue (https://github.com/keptn/keptn/issues/5692)

Reply:
will look at it :)

Reply:
Maybe this could help here: https://pkg.go.dev/text/template#Template.Option

Reply:
will add, thanks!

Reply:
we already have this Option() function used in ParseTemplate(), see https://github.com/keptn/keptn/blob/master/webhook-service/lib/template_engine.go#L16